src/Template/Element/Model/relation.twig

```diff
@@ -0,0 +1,13 @@
+<div class="input">
+    <label for="">{{__('Left types') }}</label>
+    {% for left in left_object_types|sort %}
+        <span class="tag has-background-module-{{ left }}">{{ left }}</span>
+    {% endfor %}
+</div>
+
+<div class="input">
+    <label for="">{{__('Right types') }}</label>
```

Comment:
I don't see translation for this labels. Are they missing in `.po` files?

Reply:
yes, missing translations for all the modelling part, I think.
I'd rather do a separate PR with all the translations ;)